Core/Object Arts/Dolphin/Utilities/IPHelp/IP Help API.pax

```diff
@@ -59,6 +59,7 @@ package globalAliases: (Set new
 
 package setPrerequisites: (IdentitySet new
 	add: '..\..\Base\Dolphin';
+	add: '..\..\Base\Deprecated\Dolphin Base (Deprecated)';
```

Comment:
I'll raise an issue for this. Obviously no packages should be depending on the deprecated ones.